public/js/controllers/mempool.js

```diff
@@ -61,6 +61,47 @@
         $(newRowHtml).insertBefore(rows.first())
     }
 
+    function filterVotesLastBlock(target) {
+        var best_block_hash=$(target).data("hash");
```

Comment:
Please use camelCase for client side variables, so `best_block_hash` should be `bestBlockHash`, it's simply the convention amongst the majority of open source javascript codebases.